enterprise-suite/tests/e2e/scripts/check_e2e_results.sh

```diff
@@ -0,0 +1,40 @@
+#!/usr/bin/env bash
+
+# check the e2e result file
+# the pass count should not less then $EXPECTED_PASS
+# the fail count should not greater then $EXPECTED_FAIL
+
+E2E_RESULT_FILE=results.txt
+
+EXPECTED_PASS=8
+EXPECTED_FAIL=4
+
+# RESULT is last non-empty line | remove beginning to time
+RESULT=`awk '/./{line=$0} END{print line}' $E2E_RESULT_FILE | sed -n 's/^.*:[0-9][0-9]//p'`
+TESTS=`echo  $RESULT | awk '{ print $1 }'`
+PASS=`echo  $RESULT | awk '{ print $2 }'`
+FAIL=`echo  $RESULT | awk '{ print $3 }'`
+PENDING=`echo  $RESULT | awk '{ print $4 }'`
+SKIP=`echo  $RESULT | awk '{ print $5 }'`
+
+echo "==================="
+echo "parsed test results"
+echo "TESTS=$TESTS"
+echo "PASS=$PASS"
+echo "FAIL=$FAIL"
+echo "PENDING=$PENDING"
+echo "SKIP=$SKIP"
+echo ""
+
+
+if [[ "$PASS" -lt "$EXPECTED_PASS" ]]
+then
+  echo "pass count $PASS should not less then $EXPECTED_PASS"
```

Comment:
...should not be less than...

Reply:
oh~~  this script is not really use so far.    (we might use it in the future, but not right now)
The script was designed for handling flaky test or known errors....  Now we decide to turn off all  flaky tests.